doc/whatsnew/fragments/0203.docs

```diff
@@ -0,0 +1 @@
+Added documentation for C0203 from pylint-errors
```

Comment:
No need to document this with fragments, I added the skip news label

Reply:
Removed the fragments